src/UI/Conversation.jsx

```diff
@@ -146,33 +155,58 @@ class Conversation extends Component {
 				<div className={styles.body}>
 					{
 						this.state.welcomeMessage
-							&& <Announcement message={this.state.welcomeMessage} />
+						&& <Announcement message={this.state.welcomeMessage} />
 					}
 					{
 						this.state.messages.map((message, index, array) => (
 							<React.Fragment key={message.id}>
 								{
 									this.setRenderedDate(this.getDateFromTimestamp(message.time))
-										&& <DateGroup timestamp={message.time} />
+									&& <DateGroup timestamp={message.time} />
+								}
+								{
+									message.carousel.length === 0
```

Comment:
is carousel altijd gezet? Want dan kun je op een fout komen toch of je moet een `message.carousel?.length`

Reply:
Ja die is altijd gezet. Als je de api aanroept krijg je bij elke message `carousel: []` terug. 
![image](https://github.com/parley-messaging/web-library/assets/26112463/f06f6179-def2-47c9-92ca-e705799b0984)